### PR TITLE
Sample script for finding failed jobs in View using Zowe CLI.

### PR DESCRIPTION
**Pull request by jan-podrouzek-brcm**
New sample script for finding failed jobs in View using Zowe CLI and Web Viewer.

### CODE REVIEW

**Review by adam-wolfe — APPROVED** [review]
Looks good to me. Thanks @jan-podrouzek-brcm

File: output-management/web-viewer/README.md [hunk 1]
```diff
@@ -10,3 +10,6 @@ Currently available samples:
 
 - [Downloader Application](downloader) - A sample that shows how the REST API can be used to
   implement a batch downloader for text reports into a merged local file.
+
+- [Find Failed Jobs](find-failed-jobs) - A bash script leveraging the View plug-in for Zowe CLI to
+  locate failed jobs and optinally download and search within their content.
```
Comment (bisvo01) [inline output-management/web-viewer/README.md:15]:
```suggestion
  locate failed jobs and optionally download and search within their content.
```
possibly also 
> download **them** and search within their content

File: output-management/web-viewer/find-failed-jobs/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Web Viewer - Find Failed Jobs
+
+This sample bash script uses the View Plug-in for Zowe CLI to:
+
+- Identify jobs with non-empty exception code values (xcode) archived in a View database.
+- Optionally downloads the content of the identified jobs.
+- Optionally searches for specific text within identified jobs.
+
+The script also support job filtering based on:
```
Comment (bisvo01) [inline output-management/web-viewer/find-failed-jobs/README.md:9]:
```suggestion
The script also supports job filtering based on:
```

File: output-management/web-viewer/README.md [hunk 3]
```diff
@@ -10,3 +10,6 @@ Currently available samples:
 
 - [Downloader Application](downloader) - A sample that shows how the REST API can be used to
   implement a batch downloader for text reports into a merged local file.
+
+- [Find Failed Jobs](find-failed-jobs) - A bash script leveraging the View plug-in for Zowe CLI to
```
Comment (bisvo01) [inline output-management/web-viewer/README.md:14]:
```suggestion
- [Find Failed Jobs](find-failed-jobs) - A bash script leveraging the View Plug-in for Zowe CLI to
```
consistency with [official naming](https://techdocs.broadcom.com/us/en/ca-mainframe-software/devops/ca-brightside/3-0/zowe-cli/available-cli-plug-ins/ca-view-plug-in-for-zowe-cli.html) that is already used in the other README file, possibly also applies to several places within the script itself

**Review by bisvo01 — CHANGES_REQUESTED** [review]

**Review by bisvo01 — APPROVED** [review]

**Pull request merged**